### DISCUSSION TITLE
关于应用的参数扩展问题

### DISCUSSION THREAD

**Discussion by dhloveyou** [zuihou/zuihou-admin-cloud#323] [03.想法&优化&建议]
1.比如A用户使用Ⅹ应用时设置接受Ⅹ应用短信提醒，B用户使用X应用时可以设置关闭Ⅹ应用短信提醒
2.针对应用的用户积分，余额，会员等级等信息应该放在哪里合适？放在用户和应用的关系表中是否可行？

**Comment by zuihou** [comment]
1. lamp没有内置该功能， 自己建表解决
2. def_user_application 表 设计初衷是存放每个用户的默认应用，用于登录时自动切换到该应用的。 存放你说的这些数据，可能要改默认应用查询的逻辑。
